src/app/Fake.Windows.Registry/Registry.fs

```diff
@@ -0,0 +1,112 @@
+﻿/// Contains functions which allow to read and write information from/to the registry.
+module Fake.Windows.Registry
```

Comment:
Can we add a sample to the reference documentation.
Similar to 
https://github.com/fsharp/FAKE/blob/83f79896d303c081039fd887885ca5c74c95526b/src/app/Fake.Core.ReleaseNotes/ReleaseNotes.fs#L1-L39

We would also like to add `RequireQualifiedAccess` here.